docs/client_advanced.rst

```diff
@@ -451,6 +451,24 @@ In order to specify the nameservers to when resolving the hostnames,
     conn = aiohttp.TCPConnector(resolver=resolver)
 
 
+Changing source interface
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you wish to specify where the packets should originate from, you should
+specify the interface you wish to use.
+Raises :exc:`RuntimeError` if the OS does not support this feature.
+:class:`~aiohttp.TCPConnector`::
```

Comment:
This looks like a disconnected reference that got separated from some sentence ^^^